addons/hr_expense/models/hr_expense.py

```diff
@@ -318,6 +318,10 @@ def _compute_analytic_distribution(self):
             })
             expense.analytic_distribution = distribution or expense.analytic_distribution
 
+    def _inverse_total_amount(self):
+        for expense in self:
+            expense.unit_amount = expense.total_amount_company / expense.quantity
+
```

Comment:
@JulienAlardot Didn't we say we need an `@api.onchange('total_amount')` here ?

Also, I'm wondering why we use `total_amount_company` instead of `total_amount`.
Doesn't the `hr_expense` take care of the currencies?

Reply:
True, we need that onchange
AS it is, no. Currencies for expense are a shitstorm of weird code iteratively changed back and forth. But I'm working on that in another fix

Reply:
Sounds good I have implemented the onchange